### PR TITLE
Add:Temple app project

### PR DESCRIPTION
**Pull request by abhijitramesh**

### CODE REVIEW

File: src/content/projects/Templeapp.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+slug: "Temple App"
+tagline: "Temple App is an Android app aimed for managing the finances of temples"
+description:
+  Temple App is an Android app aimed for managing the finances of temples
+cover: ./covers/Templeapp.png
+gallery:
+  - ./gallery/Templeapp-ss-1.png
+links:
+  github: https://github.com/amfoss/TempleApp
+  irc: https://gitter.im/amfoss/TempleApp?utm_source=share-link&utm_medium=link&utm_campaign=share-link
```
Comment (vibhoothi) [inline src/content/projects/Templeapp.yaml]:
```suggestion
  Glitter: https://gitter.im/amfoss/TempleApp?utm_source=share-link&utm_medium=link&utm_campaign=share-link
```

Comment (vibhoothi) [reply src/content/projects/Templeapp.yaml]: Either Change `irc` to Glitter 
Or 
Add Correct IRC Channel URL `#amfoss-android`
https://webchat.freenode.net/?channels=%23amfoss-android

**Review by vibhoothi — CHANGES_REQUESTED** [review]
Nit: Please use `Add` instead of `Added` for the commit message and do force push
Also, I have added  a suggestion, Please confirm that before merging.

File: src/content/projects/Templeapp.yaml (hunk 1 above)
Comment (abhijitramesh) [reply src/content/projects/Templeapp.yaml]: Glitter won't show the icon, nor does Gitter, The name irc would show a common message icon so I guess that would work.

Comment (vibhoothi) [reply src/content/projects/Templeapp.yaml]: Then don't point glitter in IRC

**Pull request merged**